src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/base_js.html.twig

```diff
@@ -372,7 +372,7 @@
             },
 
             createToggles: function() {
-                var toggles = document.querySelectorAll('.sf-toggle');
+                var toggles = document.querySelectorAll('a.sf-toggle');
```

Comment:
You are right. I adopt it from the navigation tabs, where the li elements don't have a class and the tag name is used. I agree, in case of the toggle button, verifying the class name is smarter.